scripts/update-changelog.sh

```diff
@@ -22,20 +22,21 @@
 set -o errexit
 set -o nounset
 set -o pipefail
+set -x
 
 GITHUB_TOKEN=${GITHUB_TOKEN:-"NO"}
```

Comment:
any github token will work?

Reply:
It should; its should for API rate limits.